landing/index.html

```diff
@@ -1,1043 +1,1146 @@
-                     <a href="#">Easy Management</a>
-                     <p>You can manage your cohort with no excuses. Add members, change user permissions, have fun!</p>
-                  </div>
-               </div>
-               
-               <div class="col-lg-4 col-md-4 col-sm-6 module">
-                  <div class="f-box wow fadeInUp" data-wow-duration="3s">
-                     <div class="f-img"><img src="images/icon4.png" alt="f4"></div>
-                     <a href="#">It's Free</a>
-                     <p>Many app's requires a charge. We believe in you and your goals, so just use it for free!</p>
+          </div>
+        </div>
+        <div class="row">
+          <div class="col-lg-4 col-md-4 col-sm-6 module">
+            <div class="f-box wow fadeInUp" data-wow-duration="1s">
+              <div class="f-img"><img src="images/icon1.png" alt="f1" /></div>
+              <a href="#">Friendly Interface</a>
+              <p>
+                User interface was created to be intuitive, easy and clean. You
+                will love it!
+              </p>
+            </div>
+          </div>
+
+          <div class="col-lg-4 col-md-4 col-sm-6 module">
+            <div class="f-box wow fadeInUp" data-wow-duration="2s">
+              <div class="f-img"><img src="images/icon2.png" alt="f2" /></div>
+              <a href="#">Well Documented</a>
+              <p>
+                Using EOC is very easy, but if you feel a bit confused, you can
+                refer to our great documentation.
+              </p>
+            </div>
+          </div>
+
+          <div class="col-lg-4 col-md-4 col-sm-6 module">
+            <div class="f-box wow fadeInUp" data-wow-duration="3s">
+              <div class="f-img"><img src="images/icon3.png" alt="f3" /></div>
+              <a href="#">Easy Management</a>
+              <p>
+                You can manage your cohort with no excuses. Add members, change
+                user permissions, have fun!
+              </p>
+            </div>
+          </div>
+
+          <div class="col-lg-4 col-md-4 col-sm-6 module">
+            <div class="f-box wow fadeInUp" data-wow-duration="3s">
+              <div class="f-img"><img src="images/icon4.png" alt="f4" /></div>
+              <a href="#">It's Free</a>
+              <p>
+                Many app's requires a charge. We believe in you and your goals,
+                so just use it for free!
+              </p>
+            </div>
+          </div>
+
+          <div class="col-lg-4 col-md-4 col-sm-6 module">
+            <div class="f-box wow fadeInUp" data-wow-duration="3s">
+              <div class="f-img"><img src="images/icon5.png" alt="f5" /></div>
+              <a href="#">Responsive Design</a>
+              <p>
+                Lorem Ipsum is simply dummy text the printing and typeseing
+                industry Lorem Ipsum has been standard.

```

Comment:
maybe something like this: use on all your devices, whenever you want and wherever you are...

Reply:
App is not responsive so I decided to leave it as is to show Lukasz and maybe he will decide to remove this section